static/js/server_events_dispatch.js

```diff
@@ -710,6 +712,16 @@ export function dispatch_normal_event(event) {
                 });
                 activity.redraw_user(event.user_id);
             }
+
+            if (event.emoji_name !== undefined) {
+                user_status.set_status_emoji({
+                    user_id: event.user_id,
+                    emoji_name: event.emoji_name,
+                    emoji_code: event.emoji_code,
+                    reaction_type: event.reaction_type,
+                });
```

Comment:
It seems like this could be written more simply using destructuring, or we could even just pass the `event`.